python/python/bystro/api/cli.py

```diff
@@ -464,6 +468,76 @@ def get_user(args: argparse.Namespace, print_result=True) -> UserProfile:
 
     return user_profile
 
+def query(args: argparse.Namespace) -> None:
+    """
+    Perform a query with the given arguments.
+
+    Parameters
+    ----------
+    args : argparse.Namespace
+        The arguments passed to the command.
+    """
+
+    state, auth_header = authenticate(args)
+
+    with open(args.postgres_config, 'r') as file:
+        postgres_config = yaml.safe_load(file)
+
+    db_host = postgres_config['host']
+    db_port = postgres_config['port']
+    db_user = postgres_config['user']
+    db_password = postgres_config['password']
+    db_database = postgres_config['database']
+
+    conn = None
+
+    try:
+        conn = psycopg2.connect(
+            host=db_host,
+            port=db_port,
+            user=db_user,
+            password=db_password,
+            database=db_database
+        )
+        cursor = conn.cursor()
+
+        if args.experiment_name:
+            sql_query = """
+                SELECT sample_id, subject_id
+                FROM user_experiments
+                WHERE experiment_name = %s
+                LIMIT %s
+            """
+            cursor.execute(sql_query, (args.experiment_name, args.top_n))
+            
+            rows = cursor.fetchall()
+            if not rows:
+                print(f"\nExperiment name '{args.experiment_name}' does not exist.")
+                return
+            
+            sample_subject_mappings = {row[0]: row[1] for row in rows}
+            print("\nSample ID => Subject ID mappings for Experiment:", args.experiment_name)
+            print(json.dumps(sample_subject_mappings, indent=4))
+            return
+
+        sql_query = "SELECT sample_id, subject_id FROM user_experiments"
```

Comment:
When the user doesn't supply an experiment_name we don't want to query user_experiments. The user hasn't given us a mapping, and subject_id's are not guaranteed to be unique across experiments.